### ISSUE TITLE
about gcn filter

### ISSUE THREAD

**Issue by durianno1** [yysijie/st-gcn#184]

**Comment by durianno1** [comment]
here,
【【【
nn.Conv2d(
        out_channels,
        out_channels,
        (temporal_kernel_size, 1),
        (1, 1),
        padding,
    )
】】】,
we use 2d cnn and use out_channel of (kernel_size,1) filter ,but i think the filter of gcn is the edge_importance which generate in  
【【【       if edge_importance_weighting:
                                                                                  self.edge_importance = nn.ParameterList([
                                                                                  nn.Parameter(torch.ones(self.A.size()))
                                                                           for i in self.st_gcn_networks
                                                                            ])
                                                                             else:
                                                                           self.edge_importance = [1] * len(self.st_gcn_networks)
                        】】】
which is the weights we need to iterate
